src/types/types.ts

```diff
@@ -526,6 +526,10 @@ export interface EdgeNetworkFee2 {
   readonly currencyPluginId: string
   readonly tokenId: EdgeTokenId
 }
+export interface EdgeNetworkFee3 {
+  readonly nativeAmount: string
+  readonly tokenId: EdgeTokenId
+}
```

Comment:
We need to get these structures sorted out, possibly requiring a separate commit / PR to clean up the mess. Let's define this:

```ts
export interface EdgeFeeAmount {
  nativeAmount: string
  tokenId: EdgeTokenId
}
```

Then we can fix:

- `EdgeTransaction.networkFees` can just be `EdgeFeeAmount[]`
- Change `EdgeSwapQuote` to use `EdgeFeeAmount & { /** @deprecated */  currencyCode: string }` instead of the old `EdgeNetworkFee`. We'll need some translation logic to upgrade / insert the `tokenId`.
- `EdgeActivationQuote` can use `EdgeFeeAmount & { /** @deprecated */ currencyPluginId: string }`. We already know the source wallet, so the `currencyPluginId` was never really needed.
- Add a `/** @deprecated use EdgeFeeAmount instead */` to both `EdgeNetworkFee` and `EdgeNetworkFee2`.